crates/composition/tests/composition/custom_scalars_basic/supergraph.graphql

```diff
@@ -0,0 +1,8 @@
+scalar Yan
+scalar Tan
+scalar Texture
+type Sheep {
+    texture: Texture
+    sheepIdx: Tethera
+}
+scalar Tethera
```

Comment:
we should maybe at some point think about the ordering of these

Reply:
This is way, way down the list, and the rendering to graphql is going to be rewritten today